.projenrc.js

```diff
@@ -16,6 +16,19 @@ const project = new typescript.TypeScriptProject({
     secret: 'GITHUB_TOKEN',
   },
   autoApproveUpgrades: true,
+  minNodeVersion: '14.17.0',
+  deps: [
+    '@aws-sdk/client-codeartifact',
+    '@aws-sdk/credential-providers',
+    '@aws-sdk/types',
+    'glob',
+    'yargs',
+    'p-queue',
+    'npm@^8',
```

Comment:
This feels annoying and like it will cause a lot of issues. 

Can we turn this into some other dependency? I.e. making it the users problem and running commands via shell?

Reply:
What is the problem? The NPM dependency?

Reply:
Yes I'm referring to the npm dependency.

Reply:
Why is it a problem?

Reply:
It's not used. Why is it even a dep?

My concern is that it's a pretty chunky dependency and I don't know if its presence in `node_modules` works well with other package managers.